services/client/src/pages/events/components/events.js

```diff
@@ -12,7 +12,7 @@ class Events extends Component {
   state = {};
 
   renderRecentEvents() {
-    const { recentEvents } = this.props;
+    const { recentEvents, location } = this.props;
```

Comment:
Should location be sent to this component or directly to `NoEvents` component? In the second case, where should I create the container?